### PR TITLE
Fixed module generator tests project template

### PR DESCRIPTION
**Pull request by TanaTau**

### CODE REVIEW

File: generators/module/templates/default/tests/_partials/SolutionX.ModuleTypeX.ModuleNameX.Tests.csproj.9.3.0.partial [hunk 1]
```diff
@@ -277,7 +279,13 @@
     <Content Include="App.config" />
   </ItemGroup>
   <ItemGroup>
-    <Analyzer Include="..\..\..\packages\xunit.analyzers.0.8.0\analyzers\dotnet\cs\xunit.analyzers.dll" />
+    <Analyzer Include="..\..\..\packages\xunit.analyzers.0.10.0\analyzers\dotnet\cs\xunit.analyzers.dll" />
```
Comment (ashishagin) [inline generators/module/templates/default/tests/_partials/SolutionX.ModuleTypeX.ModuleNameX.Tests.csproj.9.3.0.partial:282]: xunit.analyzers has been dropped from the packages.config, will this work correctly?

File: generators/module/templates/default/tests/_partials/app.config.9.3.0.partial [hunk 2]
```diff
@@ -79,6 +79,6 @@
         <assemblyIdentity name="PdfSharp" publicKeyToken="f94615aa0424f9eb" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-1.50.5147.0" newVersion="1.50.5147.0" />
       </dependentAssembly>
-	</assemblyBinding>
+	  </assemblyBinding>
```
Comment (ashishagin) [inline generators/module/templates/default/tests/_partials/app.config.9.3.0.partial:82]: Please revert this change

File: generators/module/templates/default/tests/_partials/SolutionX.ModuleTypeX.ModuleNameX.Tests.csproj.9.3.0.partial [hunk 3]
```diff
@@ -277,7 +279,13 @@
...
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\code\<%= solutionX %>.<%= moduleTypeX %>.<%= moduleNameX %>.csproj">
```
Comment (ashishagin) [inline generators/module/templates/default/tests/_partials/SolutionX.ModuleTypeX.ModuleNameX.Tests.csproj.9.3.0.partial:285]: Please double check if we need this. thx

File: generators/module/templates/default/tests/_partials/SolutionX.ModuleTypeX.ModuleNameX.Tests.csproj.9.3.0.partial [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-	  <Import Project="..\..\..\packages\xunit.runner.msbuild.2.4.1\build\net452\xunit.runner.msbuild.props" Condition="Exists('..\..\..\packages\xunit.runner.msbuild.2.4.1\build\net452\xunit.runner.msbuild.props')" />
+  <Import Project="..\..\..\packages\xunit.runner.msbuild.2.4.1\build\net452\xunit.runner.msbuild.props" Condition="Exists('..\..\..\packages\xunit.runner.msbuild.2.4.1\build\net452\xunit.runner.msbuild.props')" />
```
[the remark this answers is not in the document]
Comment (TanaTau) [reply generators/module/templates/default/tests/_partials/SolutionX.ModuleTypeX.ModuleNameX.Tests.csproj.9.3.0.partial:3]: Wrong indent

File: generators/module/templates/default/tests/_partials/SolutionX.ModuleTypeX.ModuleNameX.Tests.csproj.9.3.0.partial (hunk 1 above)
Comment (TanaTau) [reply generators/module/templates/default/tests/_partials/SolutionX.ModuleTypeX.ModuleNameX.Tests.csproj.9.3.0.partial:282]: Duplicate of xunit.analyzers has been removed from packages.config

File: generators/module/templates/default/tests/_partials/SolutionX.ModuleTypeX.ModuleNameX.Tests.csproj.9.3.0.partial (hunk 3 above)
Comment (TanaTau) [reply generators/module/templates/default/tests/_partials/SolutionX.ModuleTypeX.ModuleNameX.Tests.csproj.9.3.0.partial:285]: Removed

File: generators/module/templates/default/tests/_partials/app.config.9.3.0.partial (hunk 2 above)
Comment (TanaTau) [reply generators/module/templates/default/tests/_partials/app.config.9.3.0.partial:82]: Wrong indent

File: generators/module/templates/default/tests/_partials/SolutionX.ModuleTypeX.ModuleNameX.Tests.csproj.9.3.0.partial [hunk 5]
```diff
@@ -268,7 +268,9 @@
       <Link>license.xml</Link>
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
-    <None Include="packages.config" />
+    <Content Include="packages.config">
```
[the remark this answers is not in the document]
Comment (TanaTau) [reply generators/module/templates/default/tests/_partials/SolutionX.ModuleTypeX.ModuleNameX.Tests.csproj.9.3.0.partial:271]: Yes, it is require to restore correct packages
